### ISSUE TITLE
Compensation scenario

### ISSUE THREAD

**Issue by jbcodeforce** [ibm-cloud-architecture/eda-kc-order-cmd-mq#3]
Order C100 will have 1000 products which lead to get a voyage but not enough reefers
  - voyage V100 has orders: C01, C03 and C100
  - reefer   R01 has order C01
                R02 has order C02
                R03 has order C03

As the response to get a reefer is not-available, the order microservice put the order on-hold and compensate the voyage V100 to remove the order C100.

The Order life cycle should be created, pending, completed, cancelled, on-hold.

The UI will have 2 new pages:

 - voyage to list voyage ID -> list of order id
 - reefers to list reeferId - order ID

basically a table with two fields.

So we can demonstrate the fact that the voyage has the order and then no more. 

We can put a delay in the code of the orchestration to let the demonstration clean.
